src/components/header/_macro.njk

```diff
@@ -217,6 +240,70 @@
+                                                    {% if keyLink.url %}
+                                                        <a href="{{ keyLink.url }}" class="ons-header-nav-menu__link">{{ keyLink.text }}</a>
+                                                    {% else %}
+                                                        {{ keyLink.text }}
+                                                    {% endif %}
+                                                </p>
+                                            {% endif %}
+                                        </li>
+                                    {% endif %}
+                                {% endfor %}
+                            </ul>
+                        </div>
+                    {% endif %}
+                    {% if params.menuNavLinks.menuLinks %}
+                        <div class="ons-container">
+                            <ul class="ons-grid ons-list ons-list--bare">
+                                {% for menuLink in params.menuNavLinks.menuLinks %}
+                                    <li class="ons-grid__col ons-col-4@m ons-u-mb-no">
+                                        {% for menuItem in menuLink.menuGroup %}
+                                            {% if menuItem.heading %}
+                                                <h2 class="ons-u-fs-s--b ons-header-nav-menu__heading">{{ menuItem.heading }}</h2>
+                                            {% endif %}
+                                            {% if menuItem.subItems %}
+                                                <ul class="ons-list ons-list--bare ons-header-nav-menu__child-list">
+                                                    {% for child in menuItem.subItems %}
+                                                        {% if child.text %}
+                                                            <li class="ons-u-mb-no">
+                                                                <p class="ons-u-fs-s ons-header-nav-menu_text">
+                                                                    {% if child.url %}
+                                                                        <a href="{{ child.url }}" class="ons-header-nav-menu__link">
+                                                                            {{ child.text }}
+                                                                        </a>
+                                                                    {% else %}
+                                                                        {{ child.text }}
+                                                                    {% endif %}
+                                                                </p>
+                                                            </li>
+                                                        {% endif %}
+                                                    {% endfor %}
```

Comment:
The names here like child need updating to reflect the new param names. Otherwise it can be confusing. Might need to check that all others have been updated too